modules/log6x-6-2-0-rn.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.2/log6x-release-notes-6.2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-2-0_{context}"]
+= Logging 6.2.0 Release Notes
+
+////
+TOFIX
+This release includes link:https://access.redhat.com/errata/RHBA-2024:9038[{logging-uc} {for} Bug Fix Release 6.1.0].
```

Comment:
Probably not relevant but should all 6.1.0 entries be replaced by 6.2.0?

Reply:
good catch, fixed globally